types/dottie/index.d.ts

```diff
@@ -0,0 +1,194 @@
+// Type definitions for dottie 2.0
+// Project: https://github.com/mickhansen/dottie
+// Definitions by: Dom Armstrong <https://github.com/domarmstrong/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare const dottie: dottie.Dottie;
```

Comment:
Actually doing this causes problems with the linter due to having a method called `default`.

I worked around that by doing:
```
declare function _default(...): any;
export { _default as default };
```

But this fails the linter.

This was one of the reasons I ended up with this structure.